assets/cubyz/biomes/decorative/stone_rock.zig.zon

```diff
@@ -0,0 +1,50 @@
+.{
+	.properties = .{
+	},
+	.radius = 10,
+	.chance = 0
+	.minHeight = 50,
+	.maxHeight = 80,
+	.roughness = 1,
+	.mountains = 50,
+	.hills = 20,
+	.stoneBlock = "cubyz:stone",
+	.parentBiomes = .{
+		.{
+			.id = "cubyz:autumn/forest",
+			.chance = 2,
+		},
+		.{
+			.id = "cubyz:autumn/mixed_forest",
+			.chance = 2,
+		},
+		.{
+			.id = "cubyz:autumn/dead_forest",
+			.chance = 3,
+		},
+		.{
+			.id = "cubyz:grassland",
+			.chance = 3,
+		},
+		.{
+			.id = "cubyz:rocky_grassland",
+			.chance = 32,
+		},
+		.{
+			.id = "cubyz:taiga",
+			.chance = 3,
+		},
+		.{
+			.id = "cubyz:forest/base",
+			.chance = 4,
+		},
+		.{
+			.id = "cubyz:tundra/base",
+			.chance = 12,
+		},
+		.{
+			.id = "cubyz:tundra/patchy",
+			.chance = 6,
+		},
+	},
+}
```

Comment:
In my opinion the stone rocks are sticking into the sky like needles, instead of being ordinarily sized rocks.
I think this should be solved with something like the keepOriginalTerrain modifier for transition biomes.
I have made an issue for that: #1135